### PR TITLE
spread - support array of units in target fields

### PR DESCRIPTION
**Pull request by earthspacon**
### Description

This PR will resolve [this issue](https://github.com/effector/patronum/issues/319)

Trigger multiple units for each target field

```ts
sample({
  clock: RoomGate.open,
  target: spread({
    roomId: [getRoomFx, setSocketFx],
    userId: [setUserIdFx, userIdChanged],
  }),
})
```

### Checklist

- [x] Add tests to `src/method-name/method-name.test.ts`
- [x] Add **fork** tests to `src/method-name/method-name.fork.test.ts`
- [x] Add **type** tests to `test-typings/method-name.ts`
  - Use `// @ts-expect-error` to mark expected type error
  - `import { expectType } from 'tsd'` to check expected return type
- [x] Add documentation in `src/method-name/readme.md`
  - Add header `Patronum/MethodName`
  - Add section with overloads, if have
  - Add `Motivation`, `Formulae`, `Arguments` and `Return` sections for each overload
  - Add useful examples in `Example` section for each overload

### CODE REVIEW

**Comment by sergeysova** [comment]
Hello @earthspacon!
Thank you for the contribution.
Please, add fork tests along with type tests.

**Comment by earthspacon** [comment]
> Hello @earthspacon! Thank you for the contribution. Please, add fork tests along with type tests.

@sergeysova @AlexandrHoroshih  done

**Review by AlexandrHoroshih — APPROVED** [review]
👍

**Comment by earthspacon** [comment]
[This](https://github.com/effector/patronum/pull/333/commits/f1168e2720ba634fe19d12703e884c8e0f9363be) commit should resolve [this](https://github.com/effector/patronum/issues/291) and [this](https://github.com/effector/patronum/issues/234) issues.

But it breaks [this](https://github.com/earthspacon/patronum/blob/feat/spread-support-array-in-targets/test-typings/spread.ts#L173) test case with prepend. What do you think @sergeysova @AlexandrHoroshih

File: src/spread/spread.fork.test.ts [hunk 1]
```diff
@@ -4,64 +4,84 @@ import { spread } from './index';
...
+  const $thirdA = app.createStore('');
+  const $thirdB = app.createStore('');
+
+  const $first = app.createStore('').on(first, (_, p) => p);
+  const $second = restore(second, 0);
 
   spread({
     source,
-    targets: { first, second },
+    targets: { first, second, third: [$thirdA, $thirdB] },
```
Comment (sergeysova) [inline src/spread/spread.fork.test.ts:19]: Please, create a new tests for new feature, without modifying exists

**Comment by earthspacon** [comment]
> Or modify it with support of `.prepend((value: ExactType) => {})

yep, that's working, also with generics `.prepend<ExactType>((value) => {}) // value is ExactType`

**Review by sergeysova — APPROVED** [review]
Good job!
🧡

**Pull request merged**